### PR TITLE
fix(list): title alignment

### PR DESCRIPTION
**Pull request by jsomsanith**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
The talend-icons in list title are not aligned with the text
The input in title has no space next to the icon

**What is the new behavior?**
The icon is vertically aligned
The input after the icon has the same space than the title button

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/List/ItemTitle/ItemTitle.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// ItemTitle styles
+.item-title {
+	display: inline;
+
+	.icon {
+		width: 2rem;
+		height: 2rem;
+		vertical-align: middle;
+
+		& + input {
```
Comment (frassinier) [inline src/List/ItemTitle/ItemTitle.scss]: `&` is useless

File: src/List/ItemTitle/ItemTitle.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// ItemTitle styles
+.item-title {
+	display: inline;
+
+	.icon {
+		width: 2rem;
```
Comment (frassinier) [inline src/List/ItemTitle/ItemTitle.scss]: This should be managed by theme? Since icons should SVG now?

Comment (jsomsanith) [inline src/List/ItemTitle/ItemTitle.scss]: Removed it and replaced by #219

File: src/List/ItemTitle/ItemTitle.scss (hunk 1 above)
Comment (jsomsanith) [inline src/List/ItemTitle/ItemTitle.scss]: Removed, thanks

File: src/List/ItemTitle/ItemTitle.component.js [hunk 3]
```diff
@@ -122,7 +124,9 @@ function ItemTitle({ id, className, item, titleProps }) {
 	const value = item[key];
 	const displayMode = (displayModeKey && item[displayModeKey]) || TITLE_MODE_TEXT;
 	const iconName = iconKey && item[iconKey];
-	const icon = iconName ? <Icon className="tc-list-icon" name={iconName} /> : null;
+	const icon = iconName ?
```
Comment (frassinier) [inline src/List/ItemTitle/ItemTitle.component.js]: `IconName && <...`

File: src/List/ItemTitle/ItemTitle.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// ItemTitle styles
+.item-title {
+	display: inline;
+
+	.icon {
+		vertical-align: middle;
+
+		+ input {
+			margin-left: 15px;
```
Comment (frassinier) [inline src/List/ItemTitle/ItemTitle.scss]: You should use theme vars instead

**Pull request merged**